File: example/scalalib/testing/6-test-group-stealing/foo/test/src/GroupY9.scala
package foo
import utest._
object GroupY9 extends RandomTestsUtils {
  def tests = Tests {
    test("test1") { testGreeting("Minerva", 21) }
    test("test2") { testGreeting("Nike", 18) }
    test("test3") { testGreeting("Osiris", 19) }
    test("test4") { testGreeting("Pan", 22) }
    test("test5") { testGreeting("Qebui", 17) }
  }
} 

File: example/kotlinlib/testing/6-test-group-stealing/foo/test/src/foo/GroupY9.kt
package foo

import org.junit.jupiter.api.Test

class GroupY9 : RandomTestsUtils() {
    @Test
    @Throws(Exception::class)
    fun test1() { testGreeting("Minerva", 21) }
    
    @Test
    @Throws(Exception::class)
    fun test2() { testGreeting("Nike", 18) }
    
    @Test
    @Throws(Exception::class)
    fun test3() { testGreeting("Osiris", 19) }
    
    @Test
    @Throws(Exception::class)
    fun test4() { testGreeting("Pan", 22) }
    
    @Test
    @Throws(Exception::class)
    fun test5() { testGreeting("Qebui", 17) }
} 